plugins/source/alicloud/resources/services/bss/fetch.go

```diff
@@ -0,0 +1,174 @@
+package bss
+
+import (
+	"context"
+	"fmt"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/bssopenapi"
+	"github.com/cloudquery/cloudquery/plugins/source/alicloud/client"
+	"github.com/cloudquery/cloudquery/plugins/source/alicloud/codegen/models"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"net/http"
+	"time"
+)
+
+var (
+	_maxLimit = 100
+)
+
+// See: https://help.aliyun.com/document_detail/100400.html
+func fetchBssBillOverview(_ context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- any) error {
+	spec := meta.(*client.Client).Spec
+	cli, err := bssopenapi.NewClientWithAccessKey(spec.RegionID, spec.AccessKey, spec.SecretKey)
+	if err != nil {
+		return err
+	}
+
+	billingCycles := GetBillingCycles()
+	if spec.BillHistory > 0 {
+		billingCycles = append(GetHistoryBillingCycles(spec.BillHistory), billingCycles...)
+	}
+	for _, billingCycle := range billingCycles {
+		request := bssopenapi.CreateQueryBillOverviewRequest()
+		request.BillingCycle = billingCycle
+		response, err := cli.QueryBillOverview(request)
+		if err != nil {
+			return err
+		}
+		if response.GetHttpStatus() != http.StatusOK {
+			return fmt.Errorf("httpcode %d", response.GetHttpStatus())
```

Comment:
We may want to handle retryable errors (like 5xx and 429), either here or in a generalized function. We don't have a perfect pattern for this yet, but see the Slack plugin for an example: https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/resources/services/users/users_fetch.go#L20 and https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/client/rate_limiting.go#L13